### ISSUE TITLE
Yazar, sanatcı vs adının küçük yazılması

### ISSUE THREAD

**Issue by FatihErdem** [YouReads/YouReads#75]
Yazar, sanatçı ve benzeri adlar sol framede küçük görünebilir. Bu hali hazırda eklenti ile yapıldı. Ancak siteye de detaylı ayar olarak eklenebilir diye düşünüyorum.

**Comment by canarslan12** [comment]
Ayarlardan açılabilir bir özellik mi olacak?

**Comment by FatihErdem** [comment]
Evet. Gelismis ayarlara konabilir. Varsayilan nasil gelecek ona karar verebiliriz.

**Comment by canarslan12** [comment]
@anilari @kodbilenadam @abiyoureads sizdeyiz

**Comment by AbdurrahmanSimsek** [comment]
ben bunu eklentide çirkin buldum. secimlik olabilir ama ona laf yok

**Comment by canarslan12** [comment]
o zaman şimdilik düşük önceliğe çevirdim. yeni bir tasarım ile (*deby deki gibi) eklenebilir siteye

**Comment by FatihErdem** [comment]
Tamamdir.
